bioconda_utils/bioconda_utils-conda_build_config.yaml

```diff
@@ -1,3 +1,7 @@
+# Some of Bioconda's supported versions are described
```

Comment:
(Thanks for including those comments! :heart:)

Reply:
Happy to! =)